cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/MinTopicLeadersPerBrokerGoal.java

```diff
@@ -166,24 +166,24 @@ protected void initGoalState(ClusterModel clusterModel, OptimizationOptions opti
       throws OptimizationFailureException {
     Set<String> mustHaveTopicLeadersPerBroker = Collections.unmodifiableSet(
         Utils.getTopicNamesMatchedWithPattern(_balancingConstraint.topicsWithMinLeadersPerBrokerPattern(), clusterModel::topics));
-    _mustHaveTopicMinLeadersPerBroker = new HashMap<>();
     // populate min leaders per broker for each topic when dynamic computation is configured
+    _mustHaveTopicMinLeadersPerBroker = new HashMap<>();
```

Comment:
Nit: In corresponding comment line above, should we drop the "`when dynamic computation is configured`" part? -- i.e.`_mustHaveTopicMinLeadersPerBroker` is populated for each topic regardless of whether dynamic computation is configured or not.